lti_consumer/migrations/0006_add_on_model_config_for_lti_1p1.py

```diff
@@ -14,6 +21,12 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
+            model_name='lticonfiguration',
+            name='config_id',
+            field=models.UUIDField(default=uuid.uuid4, blank=True, null=True),
```

Comment:
Just to flag that the inspiration for this [1] does _not_ include the `blank` kwarg.
Is that an implicit default (that we're now just passing explicitly)?

- [1] https://docs.djangoproject.com/en/dev/howto/writing-migrations/#migrations-that-add-unique-fields

Reply:
I will simply remove this parameter since I don't think it clarified anything.